src/drive/web/modules/navigation/Breadcrumb/breadcrumb.styl

```
@@ -5,15 +5,30 @@
 @require '../../../../styles/coz-bar-size.styl'
 
 .fil-path-backdrop
-  flex 1 1 auto
-  width 1%
+    flex 1 1 auto
+    width 1%
+    min-width 0
+
+    &:not([override])
+        margin-right 2rem
+
+    ol
+        flex-wrap nowrap
+        min-width 0
+        overflow hidden
+        text-overflow ellipsis
+
+        li
+            &:last-child
+                    min-width 0
```

nit: indentation seems to be incorrect
```suggestion
                min-width 0
```